resources/css/bem/changelog-entry.less

```diff
@@ -16,19 +16,11 @@
     font-size: 0.7em;
   }
 
-  &__link {
-    color: @osu-colour-l1;
-
-    &:hover {
-      color: white;
-    }
-  }
-
   &__row {
     word-wrap: break-word;
 
     &--message {
-      margin-top: 2px;
+      margin-top: 8px;
```

Comment:
Message looks like this now:

<img width="545" alt="image" src="https://github.com/ppy/osu-web/assets/62069795/562d9911-a8f5-4046-9e1d-dc8101ed8c6f">

Reply:
the custom text colour for changelog-md still needs to be removed (which the original reason the link is coloured white)

Reply:
I hope I understood it correctly, but it should be fixed now.

From this:

<img width="183" alt="image" src="https://github.com/ppy/osu-web/assets/62069795/cda7197a-2b92-41b9-ba9b-394ea839cb5b">

To this:

<img width="183" alt="image" src="https://github.com/ppy/osu-web/assets/62069795/5714a551-3da2-4a08-9f20-a45df3a9f909">

(only using `changelog-md`-class)